Tests/SonarQube.TeamBuild.PreProcessor.Tests/PreProcessorTests.cs

```diff
@@ -68,7 +69,7 @@ public void PreProc_LocalPropertiesOverrideServerSettings()
                 "/d:sonar.host.url=http://host" };
 
             string configFilePath;
-            using (PreprocessTestUtils.CreateValidLegacyTeamBuildScope("tfs uri", "build uri"))
+            using (new WorkingDirectoryScope(testDir))
```

Comment:
Miscellaneous change to an existing test: the location of the generated analysis config file is related to the current directory so each test should use an unique directory to ensure the tests are independent. I've added the new WorkingDirectoryScope to temporarily change the working directory for the duration of the test.
This change fixes up an existing test in the same class. There may be other existing tests that could use this new class too.